Node2Code_Engine/Convert/ToCSharpText.cs

```diff
@@ -0,0 +1,32 @@
+﻿using BH.oM.Node2Code;
+using BH.oM.Programming;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BH.Engine.Node2Code
+{
+    public static partial class Convert
+    {
+        /***************************************************/
+        /**** Public Methods                            ****/
+        /***************************************************/
+
+        public static string ToCSharpText(this ClusterContent content)
+        {
+            if (content == null)
```

Comment:
Do we want to return any warning or note to the user that content was evaluated to be `null`?